### PR TITLE
Suggest snc/redis-bundle on composer.json

### PR DESCRIPTION
**Pull request by stephpy**
An other thing,

Why having snc_redis client by default if user has not snc/redis-bundle ? (it's only suggested and it has to be).

I guess we should show an example of usage for snc/redis-bundle but not provide a default value on `redis_client`, any thought ?

### CODE REVIEW

**Comment by jeremyFreeAgent** [comment]
I forgot to set snc/redis-bundle in require !

But if it's in suggest, yes the config default must not be snc.

Since snc/redis-bundle is the best one, I think we should use it in require.

**Comment by stephpy** [comment]
IMHO we should not. If user want to use his provider, he'll have to install snc_redis ? why ?

We could add a section on documentation to explain usage with `snc/redis-bundle`.

**Comment by jeremyFreeAgent** [comment]
Ok for me.

**Comment by shouze** [comment]
:+1: as snc/redis suggested

**Comment by shouze** [comment]
:octocat: rules them all

**Comment by stephpy** [comment]
I'll do it today, wait for commit before merge.

**Comment by jeremyFreeAgent** [comment]
Sure. Squash them.

File: README.rst [hunk 1]
```diff
@@ -1,9 +1,6 @@
 BitterBundle Documentation
 ==========================
 
-.. image:: https://secure.travis-ci.org/rezzza/BitterBundle.png?branch=master
```
Comment (jeremyFreeAgent) [inline README.rst:4]: :(

**Comment by stephpy** [comment]
Squashed. I added a branch-alias. And add `not blank` validation on Configuration.

File: README.rst (hunk 1 above)
Comment (stephpy) [inline README.rst:4]: There is no test, we'll be able to add that when we'll add tests :)

**Comment by jeremyFreeAgent** [comment]
Ok I merge it ! Thanks !

**Pull request merged**